src/vnsw/agent/contrail/contrail_agent_init.cc

```diff
@@ -46,14 +46,15 @@ void ContrailAgentInit::ProcessOptions
 void ContrailAgentInit::FactoryInit() {
     if (agent()->tsn_enabled() == false) {
         AgentObjectFactory::Register<AgentUveBase>
-            (boost::forward_adapter<boost::factory<AgentUveStats *> >(boost::factory<AgentUveStats *>()));
+            (boost::forward_adapter<boost::factory<AgentUveStats *> >
+                (boost::factory<AgentUveStats *>()));
     } else {
         AgentObjectFactory::Register<AgentUveBase>
             (boost::forward_adapter<boost::factory<AgentUve *> >(boost::factory<AgentUve *>()));
     }
     if (agent_param()->vrouter_on_nic_mode() || agent_param()->vrouter_on_host_dpdk()) {
 #ifdef _WIN32
-        LOG(DEBUG, "KSyncTcp is not supported on Windows");
+        LOG(ERROR, "KSyncTcp is not supported on Windows");
         assert(0);
```

Comment:
I remember discussion on how assert works on release mode, but I don't remember a conclusion.

Reply:
https://github.com/codilime/contrail-build/blob/windows/rules.py#L1096

Here you can see what flags I enable in `production` mode. I don't pass `NDEBUG` so asserts will work correctly.